trigger/task.go

```diff
@@ -355,10 +355,10 @@ func (t *taskTriggers) generate(sub *event.Subscription, pastTenseOverride, test
 					return nil, errors.Wrapf(err, "error getting execution task")
 				}
 				if executionTask.Details.Status == evergreen.TaskFailed {
-					if executionTask.Details.Description == evergreen.TaskDescriptionStranded {
-						shouldSkipTicket = true
-					} else {
-						shouldSkipTicket = false
+					shouldSkipTicket := executionTask.Details.Description == evergreen.TaskDescriptionStranded ||
```

Comment:
nit: should use = instead of := in this case; by using := you're creating a new `shouldSkipTicket` variable that only exists within the if statement, so the variable defined at 350 and checked at 369 will still be false. By using = you will instead assign the value to the variable defined at 350.